xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VaapiEGL.h

```diff
@@ -31,6 +33,12 @@
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
 #include <va/va.h>
+#include <va/va_drmcommon.h>
+
+#include "guilib/Geometry.h"
+#include "utils/posix/FileHandle.h"
+
+#define KODI_HAVE_VAAPI2TEXTURE VA_CHECK_VERSION(1,1,0)
```

Comment:
I see no benefit of this marco, keeping VA_CHECK_VERSION(1,1,0) saves the time required for looking up the meaning of this macro

Reply:
It's for having the conditions for enabling Vaapi2Texture coherent in the code. We don't even really know what VAAPI version this will be in at the moment.

Reply:
ok, but drop the KODI prefix. that pretends that availability is defiend by our code.

Reply:
done